### ISSUE TITLE
Modified integration saved as draft, the already deployed version cannot be deactivated

### ISSUE THREAD

**Issue by apupier** [syndesisio/syndesis-ui#1086]
- create an integration
- publish it
- modify the integration and "save as draft"

Then I have a "start integration" button but I have no more possibility to "deactivate" it.
![image](https://user-images.githubusercontent.com/1105127/31502688-20d225d6-af6e-11e7-8483-1b82d8b400aa.png)

neither from this page:
![image](https://user-images.githubusercontent.com/1105127/31502753-3e54b9c0-af6e-11e7-82b5-3fe43cf65fd8.png)

**Comment by gashcrumb** [comment]
Yeah, this definitely exposes some issues with our status management :-)

The buttons available are based on the current status of the integration.  I guess when you save it as a draft, that gets set to Draft, which probably isn't correct if there's a running instance.  We probably need to revising having `Draft` as a status on the integration object.

**Comment by rhuss** [comment]
Actually that's a consequence that we didn't do a full rework of the integration model (which is still pending here https://github.com/syndesisio/syndesis-rest/pull/493 (but very likely unusable now because of many merge conflicts) and have the latest revisions status on the integration (so its draft here, but should be of course still active). In the original story, status where only on the revisions and an integration could have a reference to a draft revision **and** and an active revision.

We have to revisit this issue very soonish as its currently one of the major pain points.
